src/rprblender/engine/render_engine.py

```diff
@@ -154,8 +155,21 @@ def _render(self):
         athena_data['Stop Time'] = datetime.datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S.%f")
         athena_data['Samples'] = self.current_sample
 
+        self.log_render_times()
         self.athena_send(athena_data)
 
+    def log_render_times(self):
+        """ Log sync and render time as minutes-seconds-milliseconds """
+        s_minutes = math.floor(self.sync_time / 60)
+        s_seconds = math.floor(self.sync_time - s_minutes * 60)
+        s_mseconds = math.ceil((self.sync_time - math.floor(self.sync_time)) * 1000)
+        log.info(f"Scene synchronization time: {s_minutes}m {s_seconds}s {s_mseconds}ms")
```

Comment:
Move converting time to str into utils module. Also some simplify calculations:
s_seconds = math.floor(self.sync_time % 60)
s_mseconds = math.floor((self.sync_time % 1.0) * 1000)